parity/cli/mod.rs

```diff
@@ -542,6 +544,7 @@ struct Mining {
 	tx_gas_limit: Option<String>,
 	tx_time_limit: Option<u64>,
 	relay_set: Option<String>,
+	min_gas_price: Option<String>,
```

Comment:
I did introduce `--min-gas-price` as a sealing/mining option and left `--gasprice` the same.  I only meant that the two have to be interchangeable in one place so I used the same type (string) initially.